connector_importer/utils/misc.py

```diff
@@ -17,7 +18,7 @@ def get_importer_for_config(backend, work_on_model, importer_config, **work_on_k
     # When using jobs, importer_config is loaded from the DB as a pure dict.
     # Make sure we always have a dotted dict.
     # FIXME: we should pass the import_type_id to the job and load it here.
```

Comment:
@simahawk is the comment above correct? AFAIS import_type_id can have few configs and at this place, we should know which exactly to apply

Reply:
Sorry, I don't get why do you need this change...

As per the fixme comment: I gotta check but we should always know the model to use at this stage.